### ISSUE TITLE
No error handling for null tiles.

### ISSUE THREAD

**Issue by reyemtm** [koopjs/koop-output-vector-tiles#3]
Is this still in development? I am looking at using Koop to add layers into AGOL using the feature server output from static geojson files. This seems like the simplest method for accomplishing this outside of loading the data into AGOL. If so I will add more issues. If not disregard.

If [geojsonVT(data).getTile(z, x, y)](https://github.com/koopjs/koop-output-vector-tiles/blob/master/index.js#L32) is null then the plugin crashes as it calls vtpbf using a null tile. A simple fix would be something like the following:
```
  if (!tile) {
    res.writeHead(204, { 'Access-Control-Allow-Origin': '*' })
    return res.end()
  }
```

**Comment by rgwozdz** [comment]
hi @reyemtm  - we would definitely like to keep moving forward on this plugin; just looking for contributors.  You should submit your fix as PR and I will happily review.

BTW, Koop sets CORS wide-open internally, so I am not sure you need the `{ 'Access-Control-Allow-Origin': '*' }`, but I haven't looked into it.

**Comment by reyemtm** [comment]
Sounds good. There are a few other updates I have in mind. I have another
issue which might be related to the core...response time is slow to get to
the output plugin...not sure how to debug that one.

**Comment by rgwozdz** [comment]
What provider are you using?

**Comment by reyemtm** [comment]
Local file geojson provider. You can check the request received log in the
output plugin. I think I am right in that the data file is passed to memory
by default?

**Comment by rgwozdz** [comment]
Yes, that provider has data cached for 10 seconds (`geojson.ttl = 10`).  After 10 seconds, it will fire getData again and re-read the file.

**Comment by reyemtm** [comment]
Oh thanks that explains what I am experiencing. I will look at adjusting
the cache timeout.

**Comment by reyemtm** [comment]
A couple more thoughts on this cache. Increasing the cache ttl fixed the
behavior I was seeing. Any reason to keep this limit low? Is there a fear
that keeping the file in cache will lead to high mem usage, or is it that
the underlying file could change? Trying to figure out the reasoning so I
can better justify my approach.

**Comment by rgwozdz** [comment]
There's no particular reason the `ttl` is to 10 seconds.  I use this provider mainly for debugging and testing koop-core, FeatureServer, and Winnow, so I haven't really scrutinize performance details.

Ideally, the `ttl` should be set by a provider-namespaced config value, so that it's not hardcoded in the `getData` function.  Feel free to submit a PR on that repo, or maybe I can squeeze in the change sometime soon.
